README.md

```diff
@@ -1,188 +1,194 @@
-- **20+ supported backends**: Ibis supports multiple query engines and DataFrame APIs.
-  Use one interface to transform with your data wherever it lives: from DataFrames in pandas to Parquet files through DuckDB to tables in BigQuery.
-- **Minimize rewrites**: Teams can often keep their Ibis code the same regardless of backend changes, like increasing or decreasing computing power, changing the number or size of their databases, or switching backends entirely.
-- **Flexibility when you need it**: When Ibis doesn't support something, it provides a way to jump directly into SQL.
-
-## Common Use Cases
-
-- **Speed up prototype to production.**
-  Scale code written and tested locally to a distributed system or cloud SQL engine with minimal rewrites.
-- **Boost performance of existing Python or pandas code.**
-  For example a general rule of thumb for pandas is "Have 5 to 10 times as much RAM as the size of your dataset".
-  When a dataset exceeds this rule using in-memory frameworks like pandas can be slow.
-  Instead, using Ibis will significantly speed up your workflows because of its deferred execution.
-  Ibis also empowers you to switch to a faster database engine, without changing much of your code.
-- **Get rid of long, error-prone, `f`-strings.**
-  Ibis provides one syntax for multiple query engines and dataframe APIs that lets you avoid learning new flavors of SQL or other framework-specific code.
-  Learn the syntax once and use that syntax anywhere.
+Ibis is the portable Python dataframe library:
 
-## Backends
-
-Ibis acts as a universal frontend to the following systems:
-
-- [Apache Arrow DataFusion](https://ibis-project.org/backends/datafusion/) (experimental)
-- [Apache Druid](https://ibis-project.org/backends/druid/) (experimental)
-- [Apache Flink](https://ibis-project.org/backends/flink)
-- [Apache Impala](https://ibis-project.org/backends/impala/)
-- [Apache PySpark](https://ibis-project.org/backends/pyspark/)
-- [BigQuery](https://ibis-project.org/backends/bigquery/)
-- [ClickHouse](https://ibis-project.org/backends/clickhouse/)
-- [Dask](https://ibis-project.org/backends/dask/)
-- [DuckDB](https://ibis-project.org/backends/duckdb/)
-- [Exasol](https://ibis-project.org/backends/exasol) (experimental)
-- [HeavyAI](https://github.com/heavyai/ibis-heavyai)
-- [MySQL](https://ibis-project.org/backends/mysql/)
-- [Oracle](https://ibis-project.org/backends/oracle/) (experimental)
-- [Pandas](https://ibis-project.org/backends/pandas/)
-- [Polars](https://ibis-project.org/backends/polars/) (experimental)
-- [PostgreSQL](https://ibis-project.org/backends/postgresql/)
-- [SQL Server](https://ibis-project.org/backends/mssql/)
-- [SQLite](https://ibis-project.org/backends/sqlite/)
-- [Snowflake](https://ibis-project.org/backends/snowflake) (experimental)
-- [Trino](https://ibis-project.org/backends/trino/) (experimental)
+- Fast local dataframes with the default DuckDB backend
```

Comment:
maybe? I'm open to changing it, but I think this serves a few purposes:

- DuckDB hype: DuckDB is awesome, people love it, and Ibis provides a Pythonic dataframe on top by default
- explain the speed -- instead of wondering how Ibis can be so fast (did it implement its own engine?) a user can realize it's from DuckDB
- hint at portability/decoupling from API from engine -- if a user grasps that the Ibis API compiles down to DuckDB SQL, they can hopefully grasp how other backends work